wp-config-pantheon.php

```diff
@@ -45,6 +45,12 @@
 define('SECURE_AUTH_SALT', $_ENV['SECURE_AUTH_SALT']);
 define('LOGGED_IN_SALT', $_ENV['LOGGED_IN_SALT']);
 define('NONCE_SALT', $_ENV['NONCE_SALT']);
+
+// When developing a WordPress Multisite locally, ensure that this constant is set.
+// This will set the Multisite variable in all Pantheon environments.
+if (getenv('FRAMEWORK') == 'wordpress_network') {
```

Comment:
Might need to add `! defined( 'WP_ALLOW_MULTISITE' ) || defined( 'WP_ALLOW_MULSTISITE' ) && ! WP_ALLOW_MULTISITE`

Reply:
```suggestion
if ( getenv('FRAMEWORK') == 'wordpress_network' && ( ! defined('WP_ALLOW_MULTISITE') || defined( 'WP_ALLOW_MULTISITE' ) && ! WP_ALLOW_MULTISITE ) ) {
```

Reply:
When this runs, the constants that a user may define in `wp-config.php` haven't been defined yet.

Reply:
The user could define it in wp-config before our include line, so this is worth checking and adding.

Reply:
Both these things are true. Ideally we'd have to advise users to update their `wp-config.php` file _or_ figure out if it's possible to declare the constant later (e.g. in the [`mu-plugin`](https://github.com/pantheon-systems/pantheon-mu-plugin/blob/main/pantheon.php) as discussed in Slack).